samples/boards/stm32/stm32l1_disco/rtc_systemclock/README.rst

```diff
@@ -0,0 +1,88 @@
+TODOOOOOOOO (if you read this I definitely forgot it)
+
+.. _stm32-stm32l1_disco-rtc_systemclock-sample:
+
+stm32l1_disco RTC system clock demo
```

Comment:
Is this specific to the STM32L1 or will it work with multiple STM32 RTC peripherals?